pkg/build/pipeline.go

```diff
@@ -140,6 +141,16 @@ func validateWith(data map[string]string, inputs map[string]config.Input) (map[s
 		if data[k] == "" {
 			data[k] = v.Default
 		}
+		if k == "expected-sha256" && data[k] != "" {
+			if !matchValidShaChars(data[k]) {
```

Comment:
can you please assert length too? As these hashes have an expected length.

Reply:
Right! Thanks :+1:

Reply:
Done :+1: